sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/SonarAnalyzer.UnitTest.csproj

```diff
@@ -159,7 +159,934 @@
     <Compile Include="Verifier.cs" />

```

Comment:
Don't commit the csproj. We use the wilcard pattern and some changes undo the wildcard and reference all files one by one which make it harder for the automation tool.